src/DynamoCore/Extensions/StartupParams.cs

```diff
@@ -22,5 +25,21 @@ public class StartupParams
         //
         // It is expected that this class will be extended in the future, so it should stay as minimal as possible.
         //
+        public IAuthProvider AuthProvider { get { return authProvider; } }
+        private IAuthProvider authProvider;
+
+        public IPathManager PathManager { get { return pathManager; } }
+        private IPathManager pathManager;
+
+        public ICustomNodeManager CustomNodeManager { get { return customNodeManager; } }
+        private ICustomNodeManager customNodeManager;
+
```

Comment:
One minor issue - let's make these backing fields `private readonly`.  This is my favorite variable declaration. :dancers: